digitalocean/resource_digitalocean_record_test.go

```diff
@@ -169,6 +169,39 @@ func TestAccDigitalOceanRecord_ExternalHostnameValue(t *testing.T) {
 	})
 }
 
+func TestAccDigitalOceanRecord_FlagsAndTag(t *testing.T) {
+	var record godo.DomainRecord
+	domain := fmt.Sprintf("foobar-test-terraform-%s.com", acctest.RandString(10))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckDigitalOceanRecordDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(
+					testAccCheckDigitalOceanRecordConfig_caa, domain),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckDigitalOceanRecordExists("digitalocean_record.foobar", &record),
+					testAccCheckDigitalOceanRecordAttributesHostname("a.foobar-test-terraform.net", &record),
```

Comment:
This test is failing for me:

```
$ make testacc TESTARGS='-run=TestAccDigitalOceanRecord_FlagsAndTag'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test $(go list ./... |grep -v 'vendor') -v -run=TestAccDigitalOceanRecord_FlagsAndTag -timeout 120m
?   	github.com/terraform-providers/terraform-provider-digitalocean	[no test files]
=== RUN   TestAccDigitalOceanRecord_FlagsAndTag
--- FAIL: TestAccDigitalOceanRecord_FlagsAndTag (2.65s)
	testing.go:434: Step 0 error: Check failed: Check 2/8 error: Bad value: expected a.foobar-test-terraform.net, got letsencrypt.org
FAIL
FAIL	github.com/terraform-providers/terraform-provider-digitalocean/digitalocean	2.652s
GNUmakefile:15: recipe for target 'testacc' failed
make: *** [testacc] Error 1
```

`&record` here is the CAA record. `testAccCheckDigitalOceanRecordAttributesHostname` seems to be a bit miss-named. It checks that the first arguement (`"a.foobar-test-terraform.net"`) matches the second argument's (a pointer to a `godo.DomainRecord`) `Data` attribute.